examples/normalizer/README.md

```diff
@@ -0,0 +1,48 @@
+# Normalizer
```

Comment:
Eh, probably doesn't really matter but this is the only markdown doc in the project whereas everything else is restructured